### ISSUE TITLE
Sentinel is fooling 

### ISSUE THREAD

**Issue by Tyloon1e** [mcmonkeyprojects/Sentinel#356]
The output of command `/version` on my server is: **This server is running Tuinity version git-Tuinity-"8056d13" (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)**
The output of command `/version citizens` on my server is: **Citizens version 2.0.27-SNAPSHOT (build 2190)**
The output of command `/version sentinel` on my server is: *Sentinel version 2.2.0-SNAPSHOT (build 376)**
A pastebin.com link to my (full!) `Sentinel/config.yml` file is: **https://pastebin.com/raw/Gd9mfUdR**
Citizens saves.yml: https://pastebin.com/raw/cuR5Y0ks

My Sentinel is behaving very... strangely. It confuses itself on its path towards a player on the move and is often running in circles. Problem is, set the speed to low and he's too slow, set it too high and he runs in circles. There's no middle ground, or is there?

Showcase, where you can see people can easily escape from the Sentinel: https://gyazo.com/356532d81ea3e29ef9dea239fe64d0a0
Players can also occupy the Sentinel for unlimited time by moving to water: https://gyazo.com/eb7d666a76ed0b07042de23f3bcb5cb4

Did I miss a configuration option? I feel like I've tried tuning everything, yet the Sentinel remains, well, slow - even with _update rate_ set to 1 tick. It doesn't feel like it updates every 1 tick.

**Comment by mcmonkey4eva** [comment]
You should definitely have update-rate at the default 10, not 1. Lowering it makes most things *less* stable, not more.

Also, you might benefit from toggling the "workaround entity chase pathfinder" config option. That option changes between two different methods of handling the chase-the-target path logic... generally set to `true` is better at dealing with fast targets.

... Also, if you want a town guard, and don't want it possible to escape, give the NPC a bow.

The water thing is a Citizens limitation, NPCs are really unsteady at pathfinding over water. It might help to try `/npc swim` and/or toggling `use-new-finder` in the Citizens config. Also probably give a step down to the waterline rather than only a ladder to get down.

**Comment by Tyloon1e** [comment]
Thanks for the tips! I'll check it out and see if I can get it to work!

**Comment by Tyloon1e** [comment]
I think the Bow thing does the job well!
Also, I noticed an error in the message when adding/removing an NPC path with /npc path.
![image](https://user-images.githubusercontent.com/52511724/100805006-2a6f2880-342e-11eb-9e10-6d0525cac775.png)

**Comment by mcmonkey4eva** [comment]
Please report that on the Citizens issues page -> https://github.com/CitizensDev/Citizens2/issues

**Comment by Tyloon1e** [comment]
Hello! A little update... I've made a few BOTs only with bows but they don't always hit people that well. Sometimes, people can actually stand right in its face, and all of its arrows somehow miss until you move enough around.

https://gyazo.com/1d01ca0a08a8c69d821590595009bf5a

Citizens config: https://pastebin.com/raw/paxVak70
Citizens saves: https://pastebin.com/raw/iPC7N3gd
Sentinel config: https://pastebin.com/raw/ftpLByGv

Not quite sure... Can I do something to guarantee annihilation every time?

**Comment by mcmonkey4eva** [comment]
It used to be required that arrows had to be spawned *in front of* an NPC to avoid the arrow magically hitting the NPC itself. This had the side effect that things very close to the NPC also couldn't get hit. (This wasn't just NPCs, it also happened with vanilla bows for players and skeleton mobs). I just tested and it looks like that requirement isn't present in modern Minecraft versions, so I've backed it up to be much much closer now (for servers in 1.16 or newer only, as I don't know in which version specifically it changed).
This patch is in build 377+ on https://ci.citizensnpcs.co/job/Sentinel/

Should fix the problem you're seeing there.
